BurnerB/PropertyProLite-CH2
# [167314882] Fix Signup Response

#### What does this PR do?
Fixes signup response

#### Description of Task to be completed
The signup response should display information about the user

#### How should this be manually tested?
* Clone this  repository https://github.com/BurnerB/PropertyProLite-CH2.git and cd into Propertypro-ch2
* checkout to branch ft-user-signup-167287789
* Run command `npm start` on your terminal
* Using [postman](https://www.getpostman.com/)
* it should repond with a response body:
```
"status": "success",
    "data": {
        "token": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpYXQiOjE1NjMzNDY3MjUsImV4cCI6MTU3MTk4NjcyNX0.15ytHPZnYJoyy4C3ohwpYg5F1ANhX0NTv-uuPr1Lf9c",
        "firstName": "John",
        "lastName": "Doe",
        "email": "johndoe6@gmail.com",
        "id": 18,
        "address": "Kigali1234"
```

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#167314882](https://www.pivotaltracker.com/story/show/167314882)

#### Screenshots (if appropriate)
![fix signup](https://user-images.githubusercontent.com/45785786/61354589-2bc0a500-a87b-11e9-846f-8447d3d3f505.png)
)
